models/base_model.py

```diff
@@ -65,3 +65,5 @@ def to_dict(self):
     def delete(self):
         """Delete current instance from storage by calling its delete method"""
         models.storage.delete(self)
+        #models.storage.save()  GOOD IDEA TO CALL SAVE HERE?
```

Comment:
I still think that calling save inside of the delete method is a bad idea. We don't call save in the new method so why call it in delete?

Reply:
understood